tokio-trace/tokio-trace-core/src/dispatcher.rs

```diff
@@ -226,6 +227,20 @@ impl Dispatch {
     pub fn drop_span(&self, id: span::Id) {
         self.subscriber.drop_span(id)
     }
+
+    /// Returns `true` if this `Dispatch` forwards to a `Subscriber` of type
+    /// `T`.
+    #[inline]
+    pub fn is<T: Any>(&self) -> bool {
```

Comment:
At a bikeshed level, I'm not sure if `is` and `downcast_ref` are perfect names. The behavior is downcast + check if the requested type is contained by the subscriber. Either way, it isn't a huge deal.

Reply:
@carllerche I also had my doubts about `is`, but I thought it was good to be consistent with existing downcasting APIs...I also considered `contains`, which seems weird.

Reply:
I suppose it could be something like `can_downcast_to`?